### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by jbisits**
All changes are related to the documentation. This is a great package that I have been using recently and in doing so came across small errors in the documentation. I did my best to follow the style of the docs but if there is something you do not like feel free to disregard!

The function [`extract`](https://rafaqz.github.io/Rasters.jl/dev/#Rasters.extract) is currently not displayed as a function so I tried to modify so it would be.

The [example](https://rafaqz.github.io/Rasters.jl/dev/#Rasters.crop) in `crop_extend.jl` does not render the figures. There was a file name error (which I corrected) but I am not sure if this will have solved the issue..

Thanks again for all your work with this great package!!

### CODE REVIEW

File: src/methods/extract.jl [hunk 1]
```diff
@@ -1,8 +1,8 @@
 """
-   extract(x, geoms; atol)
+    extract(x, geoms; atol)
```
Comment (rafaqz) [inline src/methods/extract.jl:2]: Is this space needed? Looks like it has 4 already?

**Review by rafaqz — APPROVED** [review]
Thanks, this is amazing. I rarely have time to really proof docs  changes properly, as you can tell.

**Comment by jbisits** [comment]
I can take a look at the codecov/patch tomorrow.

**Comment by rafaqz** [comment]
It's OK that's just noise really

**Pull request merged**